[dhctl] mirror: Pull only specific release, skip modules option (#8063)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE-FAQ.md

1. Pull Deckhouse images using the `dhctl mirror` command.

>    By default `dhctl mirror` pulls only the latest available patch versions for every actual Deckhouse release and the current set of officially supplied modules.

Reviewer: ```suggestion
   By default, `dhctl mirror` pulls only the latest available patch versions for every actual Deckhouse release and the current set of officially supplied modules.
```

   > Note that `--min-version` parameter will be ignored if you specify version above current rock-solid channel.

>    It is also possible to download only one specific Deckhouse release by using the `--release=X.Y.Z` flag.
>    In this case, release channels will not be used and only one specific release will be downloaded.

Reviewer: ```suggestion
   You can also pull a single specific Deckhouse release by using the `--release=X.Y.Z` flag.
   In this case, no release channels will be used and only one specific release will be pulled.
```

1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the command `dhctl mirror-modules`.

>    `dhctl mirror-modules` pulls only versions of modules available in the module release channels at the time of copying if the `--modules-filter` flag was not set.

Reviewer: ```suggestion
   `dhctl mirror-modules` pulls only the module versions available in the module release channels at the time of copying unless the `--modules-filter` flag is set.
```

   The following command will pull module images from the source described in the `ModuleSource` resource located in the `$HOME/module_source.yml` file:

   ```shell
   dhctl mirror-modules -d /tmp/d8-modules -m /tmp/module_source.yml
   ```

>    To download only a set of specific modules of specific versions, use the `--modules-filter` flag, passing the set of required modules and their versions, separated by the `;` character.

Reviewer: ```suggestion
   To download only a specific set of modules of specific versions, use the `--modules-filter` flag followed by the list of required modules and their versions separated by the `;` character.
```

   For example:

   ```shell

>    dhctl mirror-modules -d /tmp/d8-modules -m /tmp/module_source.yml --modules-filter="deckhouse-admin:v1.0.0;deckhouse-admin:v1.3.3; sds-drbd:v0.0.1"

Reviewer: --modules-filter="deckhouse-admin:v1.0.0`;`deckhouse-admin:v1.3.3`; `sds-drbd:v0.0.1"

Is there a space needed between versions? In the example above, it is there in one case and missing in the other. Or does it make no difference? (This applies to the Russian version as well.)

Author: This space is left there intentionally, to demonstrate that it is possible to use spaces between expressions
